experiments/fire.rst

```diff
@@ -0,0 +1,95 @@
+        A knife (pocket knives are great)
+
+  Warnings::
+        This is literally playing with fire.  Do it outside, in a clean well ventilated area.
+        Have a bucket of water or sand on standby, and make sure the area is away from combustible
+        materials.  A fire extinguisher isn't recommended, due to the high pressures involved being able to blow
+        flaming combustion around - better to douse in sand or water.
+        Make sure there's no large winds or other factors that might precipitate disaster.
+        If you've never made a campfire before, practice first, to get an idea of how to stack kindling
+        to make a fire, and how to properly contain a small fire.
+
+  Options::
+        Consider first demonstrating with matches, to get the safety protocol clear.
+        Also consider demonstrating some alternative methods, which are spelled out below (TODO : spell out below)
+        It can be helpful to make a fun goal at the end to stimulate interest (I recommend toasting a marshmallow!)
+
+  Method::
+        Safety first, make sure the area you're making fire in is clear from debris, away from combustible materials.
+
+        A basic bow drill is using a rapidly spinning bit of wood (the drill) to cause a huge friction in another
+        part of wood, such that an ember is formed.
+        Transferring the ember to kindling and applying air flow (blowing on it) is how the fire is actually made.
+
+        Preparing your materials::
+            Take the green stick and bend it to form a semicircular bow-like shape.
+
+            Fix the cord to either end, such that the stick can no longer go straight,
+            but not so tight that you could use it to fire an arrow.  A little slack is what you're after.
+
+            Take the dowel, and ensure the ends are rounded and not square, by trimming it a bit with the knife.
+            It doesn't have to be perfect, you're just looking to remove the right angles
+            and make it fit easier into the joints.
+
+            Take the dowel, and wrap the cord from the bow around it a turn or two.  The bow and cord
+            should grip the dowel, such that when you move the bow perpendicular
+            to the dowel, the dowel spins when fixed.
+
+            Take a piece of the dry timber (we'll call it the top holder), and cut a circular notch
+            in the middle.  This is to hold the top of the spinning dowel.  Make sure the notch is wider
+            than the dowel diameter, so it doesn't slip when under tension.
+
+            Take another piece of the dry timber, and cut a circular notch all the way through.  The notch
+            should be close to the edge, and open on the side to allow air flow.  This is the base piece,
+            where the ember will be formed.  Ensure the notch is smaller than the dowel diameter, as it'll
+            need to notch into this joint in order to create friction.
```

Comment:
I try drilling a round hole, then cutting out a cake slice  to be the point the ember collects.  I have not tried the all-the-way through idea.  WIll have to !